README.md

```diff
@@ -175,30 +178,36 @@ Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our code of conduc
 
 ## Authors
 
-- **Altinn Studio development team** - If you want to get in touch, just [create a new issue](https://github.com/Altinn/altinn-studio/issues/new).
+- **Altinn Studio development team** - If you want to get in touch, just [create a new issue][24].
 
-See also the [list of contributors](https://github.com/Altinn/altinn-studio/graphs/contributors) who participated in this project.
+See also the [list of contributors][25] who participated in this project.
 
 ## License
 
 This project is licensed under the 3-Clause BSD License - see the [LICENSE.md](LICENSE.md) file for details.
 
 [1]: https://docs.altinn.studio/
-[2]: https://dotnet.microsoft.com/download/dotnet/8.0
-[3]: https://nodejs.org
-[4]: https://git-scm.com/downloads
-[5]: https://code.visualstudio.com/Download
-[6]: https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions
-[7]: https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit
-[8]: https://www.docker.com/products/docker-desktop
-[9]: https://github.com/Altinn/altinn-studio
-[10]: http://studio.localhost
-[11]: https://reactjs.org/
-[13]: https://docs.microsoft.com/en-us/dotnet/core/
-[14]: https://docs.microsoft.com/en-us/dotnet/csharp/
-[15]: https://yarnpkg.com/
-[16]: https://www.docker.com/
-[17]: https://kubernetes.io/
-[18]: https://github.com/Altinn/altinn-studio/issues/new
-[19]: https://github.com/Altinn/altinn-studio/graphs/contributors
-[20]: https://gitea.io/
+[2]: https://docs.altinn.studio/app/getting-started/
+[3]: https://docs.altinn.studio/app/app-dev-course/
+[4]: https://github.com/Altinn/app-localtest
+[5]: https://dotnet.microsoft.com/download/dotnet/8.0
+[6]: https://nodejs.org
+[7]: https://git-scm.com/downloads
+[8]: https://code.visualstudio.com/Download
+[9]: https://code.visualstudio.com/docs/editor/extension-gallery#_workspace-recommended-extensions
+[10]: https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csdevkit
+[11]: https://www.docker.com/products/docker-desktop
+[12]: https://docs.altinn.studio/community/contributing/handbook/docker/without-docker-desktop/
+[13]: https://github.com/Altinn/altinn-studio
+[14]: https://gitea.io/
+[15]: http://studio.localhost
+[16]: https://github.com/Altinn/app-template-dotnet
+[17]: https://github.com/Altinn/altinn-studio/tree/main/frontend/testing/cypress
```

Comment:
this could also be removed regarding cypress :)

Reply:
Did you mean for all these links to be removed, or just 17?

This won't be removed yet, see above for reason.

Reply:
Yeah sry I only meant 17, but keep it for now then :)